resources/views/layout.blade.php

```diff
@@ -1,8 +1,20 @@
 @php
 use Illuminate\Support\Facades\Vite;
+use Illuminate\Foundation\Vite as ViteFoundation;
+
+$viteDataSchemeLight = new ViteFoundation();
+$viteDataSchemeLight->useStyleTagAttributes([
+'data-scheme' => 'light',
+]);
+
+$viteDataSchemeDark = new ViteFoundation();
+$viteDataSchemeDark->useStyleTagAttributes([
+'data-scheme' => 'dark',
+]);
```

Comment:
if there is a better solution to this please tell me. currently its fairly hard (impossible) to add these kind of style tag attributes with the @vite blade directive